README.md

```diff
@@ -28,6 +28,30 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
 
 ## Installation
 
+### For version 0.4.0 and above
+
+1. Click [Here](https://apps.apple.com/us/app/testflight/id899247664?mt=8) to find the TestFlight application in the AppStore
+
+1. Click “Download” to install the TestFlight application.
+
+1. After installation, open the application. A TestFlight app notification appears asking for permission to send notifications. Click “Allow” to receive notifications about app updates.
+
+1. On the “Welcome” screen, click the “Continue” button. You can now install the Woof mobile app. Follow the link below: https://testflight.apple.com/join/mGNJXIwt
```

Comment:
nit: 
Maybe make it a bit simpler
```suggestion
1. Download and install TestFlight application. [Link]((https://apps.apple.com/us/app/testflight/id899247664?mt=8) on TestFlight on App Store.

2. Open TestFlight and allow to receive notifications about app updates.

3. On the “Welcome” screen, click the “Continue” button. You can now install the Woof mobile app.
```

Reply:
In my opinion a more detailed version of the installation description is more user-friendly.